openapi-operation-validator/src/main/java/org/openapi4j/operation/validator/util/convert/MultipartConverter.java

```diff
@@ -136,6 +136,10 @@ private JsonNode convertToJsonNode(final OAIContext context,
                                      final Schema schema,
                                      final FileItemStream item,
                                      final String encoding) throws IOException {
+    
+    if (schema == null) {
+      return JsonNodeFactory.instance.nullNode();
```

Comment:
Also, it would be better to keep all field values here by using:
`TypeConverter.instance().convertPrimitive(context, schema, IOUtil.toString(item.openStream(), encoding));`
But this implies another change to type converter to handle `null` schema.

Reply:
would it be correct if `convertPrimitive` retains the field value as a text json node, when the schema is null ?

Reply:
Yes that's the idea but I didn't checked if there's any side effect.

Reply:
`convertPrimitive` already handles when schema is `null`

```
if (schema == null || value == null) {
     return JsonNodeFactory.instance.nullNode();
}
```

There is a test case `TypeConverterTest.convertPrimitiveNullNode()` and it fails when I change the implementation of `convertPrimitive` . Was this behaviour intentional ?

Reply:
I suppose it fails at line 190 with ArrayList as value.

The test should be updated to: 
```java
// no schema
    assertEquals(
      JsonNodeFactory.instance.arrayNode(),
      TypeConverter.instance().convertPrimitive(null, null, new ArrayList<>()));
```